stable/prometheus-node-exporter/values.yaml

```diff
@@ -10,6 +10,7 @@ service:
   type: ClusterIP
   port: 9100
   targetPort: 9100
+  nodePort:
```

Comment:
does this need a "0" (without the quotes) as a default value for node-port as kube-state-metrics has here: https://github.com/helm/charts/blob/master/stable/kube-state-metrics/values.yaml?